webapp/components/ListItem/ListItem.vue

```diff
@@ -1,17 +1,20 @@
 <template>
   <div id="list-item">
     <h2>{{ item.title }} - ({{ item.votes }})</h2>
-    <button @click="upvoteItem()">Upvote</button>
-    <button @click="downvoteItem()">Downvote</button>
-    <button @click="$emit('delete-item', item)">Remove</button>
+    <button v-if="isAuthenticated" @click="upvoteItem()">Upvote</button>
+    <button v-if="isAuthenticated" @click="downvoteItem()">Downvote</button>
+    <button v-if="item.isOwner" @click="$emit('delete-item', item)">
```

Comment:
⭐ For a delete and edit button that is only visible to the author of the post.

Note: I did not find an edit button.